vasppy/scripts/pimaim_to_poscar.py

```diff
@@ -38,20 +39,27 @@ def read_pimaim_restart( filename ):
     cell_matrix = lines_to_numpy_array( file_data[ -6: -3 ] )
     cell_lengths = lines_to_numpy_array( file_data[ -3: ] )
     full_cell_matrix = cell_matrix * cell_lengths
-    # TODO! need to check this with a non-orthorhombic cell
-    return( coordinates, velocities, dipoles, full_cell_matrix )
+    
+    return( coordinates, velocities, dipoles, full_cell_matrix, cell_lengths)
+
+#def get_cart_coords_from_pimaim_restart(coordinates, full_cell_matrix, cell_lengths):
```

Comment:
Can this commented out code be removed if `get_cart_coords_from_pimaim_restart` is being imported?